test/kubemark/start-kubemark.sh

```diff
@@ -202,9 +207,9 @@ EOF
 
 sed "s/##numreplicas##/${NUM_MINIONS:-10}/g" ${KUBE_ROOT}/test/kubemark/hollow-node_template.json > ${KUBE_ROOT}/test/kubemark/hollow-node.json
 sed -i'' -e "s/##project##/${PROJECT}/g" ${KUBE_ROOT}/test/kubemark/hollow-node.json
-kubectl create -f ${KUBE_ROOT}/test/kubemark/kubemark-ns.json
-kubectl create -f ${KUBECONFIG_SECRET} --namespace="kubemark"
-kubectl create -f ${KUBE_ROOT}/test/kubemark/hollow-node.json --namespace="kubemark"
+kubectl --kubeconfig=${KUBE_ROOT}/test/kubemark/kubeconfig.loc create -f ${KUBE_ROOT}/test/kubemark/kubemark-ns.json
```

Comment:
This is wrong - we don't want to start hollow nodes on a kubemark cluster, we use real cluster for it. Please revert.